hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/SerializationUtilTest.java

```diff
@@ -102,6 +108,21 @@ public void testReadWriteNullableBoolean_whenInvalid()
         deserialize(bytes);
     }
 
+    @Test
+    public void testGetInterfacesCalledVerified() {
+
+        SerializationUtil serializationUtil = mock(SerializationUtil.class);
+
+       // HashSet implements Set<E>, Cloneable, java.io.Serializable, hence using it for this test
+        Class type = new HashSet().getClass();
+        final Set<Class> interfaces = new LinkedHashSet<Class>(5);
+
+        serializationUtil.getInterfaces(type, interfaces);
+
+        verify(serializationUtil, times(1)).getInterfaces(type, interfaces);
```

Comment:
The test currently does not verify anything other than the call to `serializationUtil.getInterfaces(type, interfaces);` is made(which we call in the test method). I don't think it is useful in the current state for anything other than increasing the coverage

Reply:
Thank you, agree this was for coverage. If coverage of the method reduces in future then it will indicate to enhance the test or write a new one.

Reply:
I don't like writing a test just to artificially increase the coverage. We should assert that the method does its job, collects the interfaces to the passed collection

Reply:
Hi, I have removed this test which I will check later and I have added test for method SerializationUtil.createSerializerAdapter where I am able to assert the results. Please can you review if it looks good.